packages/cli/src/templates/express-server.ts

```diff
@@ -436,23 +436,55 @@ app.post('/api/workflows/:workflowId/execute', async (req: Request, res: Respons
 });
 
 /**
- * GET /api/memory/threads/get-by-resourceid/{resourceid}
- * @summary Get threads by resource ID
+ * GET /api/memory
+ * @summary Get memory status
  * @tags Memory
- * @param {string} resourceid.path.required - Resource identifier
+ * @return {object} 200 - Memory status
+ * @return {Error} 500 - Server error
+ */
+app.get('/api/memory', async (_req: Request, res: Response) => {
+  try {
+    const memory = mastra.memory;
+
+    if (!memory) {
+      res.json({ result: false });
+      return;
+    }
+    res.json({ result: true });
```

Comment:
should we not return memory here ?

Reply:
the aim was to just confirm the status of memory
but yeah, can just return memory there, if dev wants to use it for anything
will do that

Reply:
oh i see, maybe we should do /api/memory/status then ?